packages/remix-dev/compiler.ts

```diff
@@ -1,4 +1,4 @@
-import { promises as fsp } from "fs";
+import * as fse from "fs-extra";
```

Comment:
Note: this change isn't necessary anymore since I moved the plugin into a separate file. Not using anything from `fs-extra` in this file.